PasswordVault.Desktop.Winforms/Logic/DesktopServiceWrapper/DesktopServiceWrapper.cs

```diff
@@ -59,34 +58,13 @@ public AuthenticateResult Login(string username, string password)
 
             if (!IsLoggedIn())
             {
-                // Temporarily grab the user from database to retreive the encryption parameters
-                User tempUser = _userService.GetUserByUsername(username);
-
-                if (tempUser != null)
-                {
-                    // If the user is valid, store the encryption parameters for the session
-                    _encryptionParameters = new EncryptionServiceParameters(
-                        (EncryptionService)tempUser.PasswordEncryptionService.Value,
-                        new EncryptionSizes(
-                        tempUser.PasswordIterations.Value,
-                        tempUser.PasswordBlockSize.Value,
-                        tempUser.PasswordKeySize.Value));
-                }
-
-                // Clear out the temp user from memory
-                tempUser = new User(false);
-
-                AuthenticateReturn authenticateResult = _authenticationService.Authenticate(username, password, _encryptionParameters);
+                AuthenticateReturn authenticateResult = _authenticationService.Authenticate(username, password, _settings.DefaultEncryptionParameters);
```

Comment:
Add some comments as to why default parameters are passed in and how that works.